doc/howto/snap.md

```diff
@@ -171,6 +171,21 @@ Example:
 sudo enterprise-store override lxd stable=25846
 ```
 
+(howto-snap-change)=
+## Change the snap channel
+
+While it is possible to change the channel used at installation, proceed with caution.
+
+You can upgrade (move to a newer {ref}`track <ref-snap-tracks>`, such as from {{current_lts_track}} to {{current_feature_track}}), as well as move to different {ref}`risk level <ref-snap-risk>` with the same track. However, downgrading (moving to a channel with an older track, such as from {{current_feature_track}} to {{current_lts_track}}) is neither recommended nor supported, as breaking changes can exist between major versions.
```

Comment:
> as well as move to different {ref}`risk level <ref-snap-risk>` with the same track.

Sometimes this isn't true when switching risk levels in `latest/*` or the current feature track, as new DB schema changes may have landed in `{latest,6}/{edge,candidate}` that are not in `/stable` that would prevent switch back to `/stable`.